src/Validation/Validator.php

```diff
@@ -30,6 +30,12 @@
  */
 class Validator implements ArrayAccess, IteratorAggregate, Countable
 {
+    /**
+     * Used to flag nested rules created with addNested() and addNestedMany()
+     *
+     * @var string
+     */
+    const NESTED = '_nested';
```

Comment:
Ok, I hope it is documented in ghe conventions chapter (e.g to avoid underscore prefixed sting identifiers).
Didn't that change for formhelpers options, too? Thanks!